appcelerator-archive/amp
# fix #320 - Kapacitor alerts are less noisy

how to test:
- export AMP_SLACK_ALERTS_CHANNEL="@you_login"  (or #channel)
- export AMP_SLACK_ALERTS_WEBHOOK_URL=https://hooks.slack.com/services/xxxx/cccc/zzz
- swarm start
- monitor your Slack app, notification should appear in the slackbot conversation
- you should have a few notification, but not a constant flow
